lib/cell.rb

```diff
@@ -25,18 +25,22 @@ def fired_upon?
   def fire_upon
     @fired_upon = true
     if empty? == false
-      ship.hit
+      @ship.hit
```

Comment:
good catch on fixing a scope issue and maintaining consistency in code.

Reply:
Not sure if it was necessary, but was debugging altogether and kept this @ in there.